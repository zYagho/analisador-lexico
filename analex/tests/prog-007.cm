int a;
int b[10];
int c[3][5];

b = 10

int func(int x, int y){
  int res;
  res = x + y;
  return(res);
}

int main(){
  a = input();
  b = input();

  b[0] = a;
  b[1] = b;
  c[0][1] = func(a,b);

  output(c[3]);
  return(0);
}
