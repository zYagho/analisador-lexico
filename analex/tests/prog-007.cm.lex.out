INT
ID
SEMICOLON
INT
ID
LBRACKETS
NUMBER
RBRACKETS
SEMICOLON
INT
ID
LBRACKETS
NUMBER
RBRACKETS
LBRACKETS
NUMBER
RBRACKETS
SEMICOLON
ID
ATTRIBUTION
NUMBER
SEMICOLON
INT
ID
LPAREN
INT
ID
COMMA
INT
ID
RPAREN
LBRACES
INT
ID
SEMICOLON
ID
ATTRIBUTION
ID
PLUS
ID
SEMICOLON
RETURN
LPAREN
ID
RPAREN
SEMICOLON
RBRACES
INT
ID
LPAREN
RPAREN
LBRACES
ID
ATTRIBUTION
ID
LPAREN
RPAREN
SEMICOLON
ID
ATTRIBUTION
ID
LPAREN
RPAREN
SEMICOLON
ID
LBRACKETS
NUMBER
RBRACKETS
ATTRIBUTION
ID
SEMICOLON
ID
LBRACKETS
NUMBER
RBRACKETS
ATTRIBUTION
ID
SEMICOLON
ID
LBRACKETS
NUMBER
RBRACKETS
LBRACKETS
NUMBER
RBRACKETS
ATTRIBUTION
ID
LPAREN
ID
COMMA
ID
RPAREN
SEMICOLON
ID
LPAREN
ID
LBRACKETS
NUMBER
RBRACKETS
RPAREN
SEMICOLON
RETURN
LPAREN
NUMBER
RPAREN
SEMICOLON
RBRACES
